EnvelopeItem/InteropMessageStamp.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Enqueue\MessengerAdapter\EnvelopeItem;
+
+use Interop\Queue\Message;
+use Symfony\Component\Messenger\Stamp\StampInterface;
+
+final class InteropMessageStamp implements StampInterface
```

Comment:
Make this implement `NonSendableStampInterface`. These stamps exist just so that we can `ack()` and `reject()`. But, for example, if there is a retry, this stamp should not be serialized and sent to the transport.

Reply:
Wonderful!